test/robustness/overlay/areal_areal/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ foreach(item IN ITEMS
     recursive_polygons
     star_comb
     #ticket_9081
+    #random_integer_grid
```

Comment:
It should be `random_integer_grids` (with `s`) and not commented.

BTW, about the name, it's now really targeting integer. Suggestion: `random_bitset_grids` instead.